source/ada/lsp-ada_handlers.adb

```diff
@@ -676,6 +680,17 @@ package body LSP.Ada_Handlers is
         (Is_Set => True,
          Value => (Is_Boolean => True, Bool => True));
 
+      if Value.capabilities.workspace.workspaceEdit.documentChanges.Is_Set then
```

Comment:
This `if` could be rewritten a bit shorter using `True` value for `Optional_Boolean` type:
```ada
Self.Versioned_Documents := 
  Value.capabilities.workspace.workspaceEdit.documentChanges = True;
```